tests/acceptance/blockchain/conditions/conftest.py

```diff
@@ -199,7 +205,9 @@ def subscription_manager_get_policy_zeroized_policy_struct_condition(
     )
     condition = ContractCondition(
         contract_address=subscription_manager.contract.address,
-        function_abi=[subscription_manager.contract.find_functions_by_name('getPolicy')[0].abi],
+        function_abi=subscription_manager.contract.find_functions_by_name("getPolicy")[
+            0
```

Comment:
Sometimes blackened code format is less readable... 🤔

Reply:
Same as above, can be simplified by using `get_function_by_name` (see PR https://github.com/theref/nucypher/pull/1).